engine/src/test/java/org/hibernate/validator/test/internal/metadata/location/ConstraintLocationTest.java

```diff
@@ -38,9 +38,33 @@ public void two_constraint_locations_for_the_same_member_should_be_equal() throw
 		assertEquals( location1, location2, "Two constraint locations for the same type should be equal" );
 	}
 
+	@Test
+	@TestForIssue(jiraKey = "HV-1534")
+	public void owning_class_of_member_is_declaring_class() throws Exception {
+		Method getter = Bar.class.getMethod( "getBar" );
+		ConstraintLocation location = ConstraintLocation.forGetter( getter );
+
+		assertEquals( location.getDeclaringClass(), Foo.class, "The owning class of a member should be the declaring class" );
+	}
+
+	@Test
+	@TestForIssue(jiraKey = "HV-1534")
+	public void bean_class_is_declaring_class() throws Exception {
+		Method getter = Bar.class.getMethod( "getBar" );
+		ConstraintLocation location = ConstraintLocation.forGetterOfClass( getter, Bar.class );
+
+		assertEquals( location.getDeclaringClass(), Bar.class, "The bean class should be the declaring class" );
```

Comment:
You can remove these two tests, they are useless and it's not a good idea to add tests for such trivial things.